### PR TITLE
Vgmstream r1050-3280-gba405509

### PR DESCRIPTION
**Pull request by xxyzz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by equal-l2** [comment]
Will you add `license: "ISC"` to remove "missing license" label?

File: Formula/vgmstream.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://hcs64.com/vgmstream.html"
-  url "https://github.com/losnoco/vgmstream/archive/r1050-3264-g86fbfffd.tar.gz"
-  version "r1050-3264-g86fbfffd"
-  sha256 "c7cb968c734b02dcfce46a3abf3486be494a599c238d77c1103c302e91763d11"
+  version "r1050-3280-gba405509"
+  url "https://github.com/losnoco/vgmstream/archive/#{version}.tar.gz"
```
Comment (SMillerDev) [inline Formula/vgmstream.rb:5]:
```suggestion
  url "https://github.com/losnoco/vgmstream/archive/r1050-3280-gba405509.tar.gz"
  version "r1050-3280-gba405509"
```

**Comment by xxyzz** [comment]
I'm not sure how homebrew build this thing but it need to set `BUILD_AUDACIOUS` and `USE_JANSSON` to `OFF` if these two are not installed. Like `cmake -D BUILD_AUDACIOUS:BOOL=OFF -D USE_JANSSON:BOOL=OFF  .`

**Comment by xxyzz** [comment]
Eh.. I don't know how to build it from a release `.tar.gz` file. I guess `cmake` needs a git repo(.git folder): https://github.com/losnoco/vgmstream/issues/721

File: Formula/vgmstream.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://hcs64.com/vgmstream.html"
-  url "https://github.com/losnoco/vgmstream/archive/r1050-3280-gba405509.tar.gz"
-  version "r1050-3280-gba405509"
-  sha256 "ab5c900af26f642ffcf96e95eb83e79722e07751f861afe3cd8acf5239394f1d"
+  url "https://github.com/losnoco/vgmstream.git",
+      tag: "r1050-3280-gba405509"
```
Comment (Rylan12) [inline Formula/vgmstream.rb:5]:
```suggestion
  url "https://github.com/losnoco/vgmstream.git",
      tag:      "r1050-3280-gba405509",
      revision: "xxxxxxxxxxx"
```

For git repos, we require specifying both a `tag` and a `revision` which points to the commit that it tag points to.

File: Formula/vgmstream.rb [hunk 3]
```diff
@@ -15,21 +14,21 @@ class Vgmstream < Formula
...
   depends_on "libvorbis"
   depends_on "mpg123"
-  depends_on "ffmpeg"
 
   def install
     system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", "."
     system "make", "vgmstream_cli"
     system "make", "vgmstream123"
-    bin.install "cli/vgmstream-cli"
+    bin.install "cli/vgmstream_cli"
```
Comment (Rylan12) [inline Formula/vgmstream.rb:26]: Will it be surprising to users if the binary is renamed from `vgmstream-cli` to `vgmstream_cli`?

Comment (xxyzz) [reply Formula/vgmstream.rb:27]: The purpose of adding `cmake` command is to enable FFmpeg to supports more codecs but the binary name will change. The document of vgmstream mentioned another build method(https://github.com/losnoco/vgmstream/blob/master/doc/BUILD.md#audacious-plugin) doesn't rename the binary but I don't know how to use it correctly(disable audacious and enable ffmpeg).

Comment (SMillerDev) [reply Formula/vgmstream.rb:27]: you should make a symlink `bin.install_symlink "vgmstream_cli" => "vgmstream-cli"`

Comment (xxyzz) [reply Formula/vgmstream.rb:27]: Thanks!

**Comment by SMillerDev** [comment]
Can you squash all your commits into one?

**Review by SMillerDev — APPROVED** [review]
Thanks @xxyzz  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**